### DISCUSSION TITLE
Book chapters

### DISCUSSION THREAD

**Discussion by rastaman111** [TortugaPower/BookPlayer#676] [Q&A]
Hey!  where can i find a book with chapters?  If you have a test book, then I would be glad if you share with it.

**Comment by luwimi** [comment]
Podcasts often use Chapter Marks. Often also with images. One Example is the come-fly-with-us-Podcast, e.g. Episode 105 https://www.comeflywithus.de/2023/03/05/cfwu-105-no-animals-were-harmed/
This would also be useful for implementing chapter images.
